apps/callscreen/js/calls_handler.js

```diff
@@ -243,15 +243,15 @@ var CallsHandler = (function callsHandler() {
   }
 
   /**
-   * Play the busy tone in response to the corresponding error being triggered
-   * at the end of a call. Once the tone has finished this will also
-   * automatically close the callscreen.
+   * Play the busy tone in response to a disconnected event with the
+   * appropriate reason set. Once the tone has finished this will also
+   * automatically close the callscreen if no more calls are present.
    *
-   * @param evt {Object} The event delivered in the TelephonyCall.onerror
-   *        event-handler.
+   * @param evt {Object} The event delivered in the
+   *        TelephonyCall.ondisconnected event-handler.
    */
-  function handleBusyErrorAndPlayTone(evt) {
-    if (evt.call.error.name === 'BusyError') {
+  function playToneWhenBusy(evt) {
```

Comment:
I haven't removed it, just renamed to ``playToneWhenBusy`` to make the intent clearer.